### DISCUSSION TITLE
Is it possible to notify a conversation started on website to my telegram account?

### DISCUSSION THREAD

**Discussion by Yramgar** [chatwoot/chatwoot#5572] [Q&A] [answered]
I have a "website" channel which is working fine, but the notifications are not foolproof. I would like to know if it is possible to be notified by Telegram when a new conversation is started or when new messages arrive. Is it possible? How to do it? Thank you.

**Comment by sojan-official** [comment, marked as answer]
@Yramgar  we don't have a feature for this. But you can set up a cloud function listening to the chatwoot webhooks and then trigger a telegram notification based on that.
